plists/v1parser.py

```diff
@@ -28,10 +27,11 @@ class Parser(object):
 
         obj = Parser().parse(<string_or_stream>)
     """
-    def parseFile(self, path):
-        return self.parse(open(path))
 
-    def parse(self, string_or_stream):
+    def parseFile(self, path, untokenize_=False):
```

Comment:
Hmm just a quick one (I should have done this earlier :) ).   Can you add comments to this method as well - I wasnt sure of the purpose of the untokenizing.  Is it so that if you have a plist stream then the start of the next plist object can be reliably obtained later on?